File: Programming EF Code First/chapter3/Chapter 3 VB/Fluent/Model/Destination.vb
Namespace Model
  Public Class Destination
	Public Property DestinationId() As Integer
	Public Property Name() As String
	Public Property Country() As String
	Public Property Description() As String
	Public Property Photo() As Byte()
	Public Property Lodgings() As List(Of Lodging)
  End Class
End Namespace

File: Programming EF Code First/chapter2/Chapter2/BAGA_FluentAPI/Model/Destination.cs
using System.Collections.Generic;

namespace Model
{
  public class Destination
  {
    public int DestinationId { get; set; }
    public string Name { get; set; }
    public string Country { get; set; }
    public string Description { get; set; }
    public byte[] Photo { get; set; }

    public List<Lodging> Lodgings { get; set; }
  }
}